app/server.go

```diff
@@ -752,8 +753,8 @@ func (s *Server) Channels() *Channels {
 
 // Return Database type (postgres or mysql) and current version of Mattermost
 func (s *Server) DatabaseTypeAndMattermostVersion() (string, string) {
```

Comment:
Looks like `generateSupportPacketYaml` uses this method to embed `the database (mysql/postgres + current Mattermost version)`. Since I believe it's possible for the database schema version and Mattermost version to diverge, e.g. because the server was reverted while the schema was left in tact, should we add a component to the packet that also embeds the current server version?